tooling/autodispose-rx-error-prone-checker/build.gradle

```diff
@@ -0,0 +1,70 @@
+plugins {
+  id "net.ltgt.errorprone" version "0.0.13"
+  id "com.github.johnrengelman.shadow" version "2.0.1"
+  id "com.github.johnrengelman.plugin-shadow" version "2.0.0"
```

Comment:
Remove this line

Reply:
https://github.com/uber/AutoDispose/pull/156/commits/05a3a34b3778da0031e5dfa0161256f224ed6cdf